src/clr-angular/utils/drag-and-drop/drag-and-drop.module.ts

```diff
@@ -0,0 +1,13 @@
+/*
+ * Copyright (c) 2016-2018 VMware, Inc. All Rights Reserved.
+ * This software is released under MIT license.
+ * The full license information can be found in LICENSE in the root directory of this project.
+ */
+
+import {CommonModule} from "@angular/common";
+import {NgModule} from "@angular/core";
+
+import {ClrDragAndDropEventBus} from "./providers/drag-and-drop-event-bus";
+
+@NgModule({imports: [CommonModule], providers: [ClrDragAndDropEventBus]})
+export class ClrDragAndDropModule {}
```

Comment:
A general question for naming, `ClrDrag*` seems more concise. Everything revolves around the HTML `DragEvent`s. Open for debate, but feels better to me.

Reply:
I think ClrDragAndDrop* makes more sense for the classes/modules that controls both draggable and droppable. Classes that are only involved in the draggable and drag events will be named ClrDrag*.

Reply:
I can live with that.